01_intro-r.qmd

```diff
@@ -54,68 +64,85 @@ R comes from the S programming language and S-PLUS system. In addition to offeri
 
 Source: "Stages in the Evolution of S" via [Roger Peng](https://simplystatistics.org/posts/2018-07-12-use-r-keynote-2018/)
 
-# RStudio
+## RStudio
 
-RStudio is a free, open-source integrated development environment (IDE) that runs on top of R. In practice, R users almost exclusively open RStudio and rarely directly open R.
+RStudio is a free, open-source integrated development environment (IDE) that runs on top of R. In practice, R users almost exclusively open RStudio and rarely directly open R. When we say IDE, we mean a piece of software where you can write, or develop, code in an efficient way. 
 
 RStudio is developed by a for-profit company called [RStudio](https://rstudio.com/). RStudio, the company, employs some of the R community’s most prolific, open-source developers and creates many open-source tools and resources.
 
 While R code can be written in any text editor, the RStudio IDE is a powerful tool with a console, syntax-highlighting, and debugging tools. [The RStudio IDE cheatsheet](https://www.rstudio.com/resources/cheatsheets/) outlines some of the power of RStudio.
 
-![](../images/rstudio-ide.png){width=400}
+![](images/rstudio-ide.png){width=400}
 
-# RStudio Console
+## RStudio Console
 
-![](../images/rstudio-console.png){width=400}
+![](images/rstudio-console.png){width=400}
 
 The RStudio Console contains the R command line and R output from previously submitted R code. 
 
 Code can be submitted by typing to the right of the last blue `>` and the hitting enter. 
 
-### `r kableExtra::text_spec("Exercise", color = "#1696d2")`
+::: callout
 
-R has all of the functionality of a basic calculator. Let's run some simple calculations with addition (`+`), subtraction (`-`), multiplication (`*`), division (`/`), exponentiation (^), and grouping (()). 
+#### [`r paste("Exercise", exercise_number)`]{style="color:#1696d2;"}
 
-$$\cdot\cdot\cdot$$
+```{r}
+#| echo: false
+
+exercise_number <- exercise_number + 1
+
+```
 
-# .R Scripts
+R has all of the functionality of a basic calculator. Let's run some simple calculations with addition (`+`), subtraction (`-`), multiplication (`*`), division (`/`), exponentiation (^), and grouping (`()`). 
 
-![](../images/rstudio-editor.png){width=400}
+:::
+
+## .R Scripts
+
+![](images/rstudio-editor.png){width=400}
```

Comment:
I think all of these images need to be bigger.

Reply:
agreed